MapboxNavigation/InstructionsCardViewController.swift

```diff
@@ -177,12 +183,16 @@ open class InstructionsCardViewController: UIViewController {
     
     func snapToIndexPath(_ indexPath: IndexPath) {
         guard let itemCount = steps?.count, itemCount >= 0 && indexPath.row < itemCount else { return }
-        instructionsCardLayout.collectionView?.scrollToItem(at: indexPath, at: .left, animated: true)
+        instructionsCardLayout.collectionView?.isPagingEnabled = false
```

Comment:
Xcode 12.1 (iOS 14.1) and Xcode 12.2 (iOS 14.2) are available. We should test whether scrolling issue was fixed in those versions. If not, I think it's acceptable to use such workaround. It should be thoroughly tested though: in landscape and portrait modes during rotation for iPhones (with/without notches) and iPads. 

There should also be `@available` check to run this code only for iOS 14 and leave it unchanged for iOS 13 and lower.

Reply:
Sounds good. I will test with Xcode 12.2. I tested with Xcode 12.1 and it wasn't fixed.

Will add the check!

Reply:
Wasn't fixed with Xcode 12.2 (iOS 14.2)